backend/utils/file_validator.py

```
@@ -23,6 +23,7 @@ class FileValidator:
     LATEX_EXTENSIONS = {'.tex'}
     AUDIO_EXTENSIONS = {'.ogg', '.opus', '.mp3', '.wav', '.m4a', '.aac', '.flac'}
     IMAGE_EXTENSIONS = {'.jpg', '.jpeg', '.png', '.bmp', '.tiff', '.gif', '.webp'}
+    VIDEO_EXTENSIONS = {'.mp4', '.mov', '.mkv', '.webm', '.avi', '.m4v'}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)